config/standard/translations/messages-en.properties

```diff
@@ -1,3 +1,6 @@
+messages.d.validation.delivery_code = {{\#delivery_code}}The delivery code was not understood. Please use code 'F' for facility, 'S' for homebirth with Skilled Birth Attendant, 'NS' for birth without SBA.{{/delivery_code}}{{^delivery_code}}The delivery code was missing. Please use code 'F' for facility, 'S' for homebirth with Skilled Birth Attendant, 'NS' for birth with no skilled attendant.{{/delivery_code}}
+messages.c.validation.patient_name = {{\#patient_name}}Registration format is incorrect. Start with C followed by space, age in weeks followed by a space, and the child's name (maximum of 30 characters).{{/patient_name}}{{^patient_name}}.The registration format is missing the name. Start with C followed by space, age in weeks followed by a space, and the child's name{{/patient_name}}.
+messages.n.validation.patient_name = {{\#patient_name}}The registration format is incorrect, ensure the message starts with N followed by space and the person's name (maximum of 30 characters).{{/patient_name}}{{^patient_name}}The registration format is incorrect, ensure the message starts with N followed by space and the person's name.{{/patient_name}}.
```

Comment:
Where did these come from? :thinking:

Reply:
https://github.com/medic/cht-core/blob/master/ddocs/medic/_attachments/translations/messages-en.properties#L760 :slightly_smiling_face:

Reply:
Ok. They shouldn't be listed in both files. The ones in `_attachments` are automatically deployed for each instance anyway. From the look of these keys they should _only_ be specified in the `config/standard/translations/` file, so I guess remove them from the `ddocs/medic/_attachments/translations` one?

Reply:
Yes I added the 'en' translations because these keys are used in [messages-sw.properties](https://github.com/medic/cht-core/blob/125e1c8b3c9c9f4155a0e3572d8f734b9a0c0f61/config/standard/translations/messages-sw.properties#L1-L3) and have placeholders that need to much the base translations, so as you suggest I will remove them in both 'en' and 'sw' files, I checked the 'sw' texts and there is no difference with the ones in `_attachments/translations/messages-sw.properties`.

Reply:
Done.